sql/core/src/test/resources/sql-tests/results/pgSQL/strings.sql.out

```diff
@@ -0,0 +1,718 @@
+-- Automatically generated by SQLQueryTestSuite
+-- Number of queries: 88
+
+
+-- !query 0
+SELECT 'first line'
+' - next line'
+	' - third line'
+	AS `Three lines to one`
+-- !query 0 schema
+struct<Three lines to one:string>
+-- !query 0 output
+first line - next line - third line
+
+
+-- !query 1
+SELECT 'first line'
+' - next line' /* this comment is not allowed here */
+' - third line'
+	AS `Illegal comment within continuation`
+-- !query 1 schema
+struct<Illegal comment within continuation:string>
+-- !query 1 output
+first line - next line - third line
+
+
+-- !query 2
+SELECT binary('\\xDeAdBeEf')
+-- !query 2 schema
+struct<CAST(\xDeAdBeEf AS BINARY):binary>
+-- !query 2 output
+\xDeAdBeEf
```

Comment:
different output? https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/expected/strings.out#L160

Reply:
Maybe the reason is output style:
```sql
postgres=# SET bytea_output TO hex;
SET
postgres=# SELECT E'\\xDeAdBeEf'::bytea;
   bytea
------------
 \xdeadbeef
(1 row)

postgres=# SET bytea_output TO escape;
SET
postgres=# SELECT E'\\xDeAdBeEf'::bytea;
      bytea
------------------
 \336\255\276\357
(1 row)
```